spec/requests/shopping_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe 'Shoppings', type: :request do
+  describe 'GET /index' do
+    it 'returns http success' do
+      get '/shopping/index'
+      expect(response).to have_http_status(:success)
+    end
+  end
```

Comment:
- [ ] As required for the project, kindly ensure that all your tests are passing as there some of them that are failing.
![image](https://github.com/RileyManda/recipe-app/assets/83042742/cf72c0cf-b737-422d-a495-218688136429)